src/lib/times.js

```diff
@@ -41,6 +42,10 @@ export function timeForId(time, id) {
       rv = rv.day(6).hour(9).minute(0);
       text = getLocalizedDateTime(rv, 'short_date_time');
       break;
+    case 'weekday':
+      rv = rv.isoWeekday(8).hour(9).minute(0);
```

Comment:
We're already using `.day` everywhere, so let's keep consistent with that, please. (Also up in the `hideif` functions.)

Reply:
I used `isoWeekday` since it goes from 1 = Monday to 7 = Sunday. `day` goes from 0 = Sunday to 6 = Saturday. So to get the next Monday, I would need to differentiate and use `.day(1)` if it is Sunday, and `.day(8)` on any other day.

Similar in `hideif`, where I can just use `>` to get the weekend instead of comparing to `0` or `6`